actions/release-candidate/dist/post/index.js

```diff
@@ -125551,16 +125550,29 @@ async function run() {
 			}
 		}
 
-		if (publish) {
+		if (do_publish) {
 			await exec("svn", ["add", artifact_dir]);
-			await exec("svn", ["commit", "--username", svn_username, "--password", svn_password, "--message", `Stage ${ project_name } ${ release_version }`, artifact_dir]);
+			await exec("svn", ["commit", "--config-dir", svn_config_dir, "--message", `Stage ${ project_name } ${ release_version }`, artifact_dir]);
+			fs.rmSync(svn_config_dir, { recursive: true, force: true });
```

Comment:
I dont' think this is necessary. The plaintext passwords are already easily gotten in other stages of the workflow if a malicious user wants them. And after this, the entirey container should be destroyed